integrations/server/test_norostat.py

```diff
@@ -0,0 +1,59 @@
+# first party
+from delphi.epidata.common.integration_test_base_class import BasicIntegrationTest
+
+
+class NorostatTest(BasicIntegrationTest):
+    """Basic integration tests for norostat endpint."""
+
+    def setUp(self) -> None:
+        """Perform per-test setup."""
+
+        create_norostat_point_diffs = """
```

Comment:
is this the same `CREATE TABLE` definition as in `test_meta_norostat.py`?  you could put the tests from both files into just one file to reduce repetition if you want!